lib/improver/tests/utilities/cube_manipulation/test__equalise_coord_bounds.py

```diff
@@ -138,6 +148,61 @@ def test_mismatched_bounds(self):
         with self.assertRaisesRegex(ValueError, msg):
             _equalise_coord_bounds(result)
 
+    def test_with_forecast_period_bounds(self):
+        """ Test that inputs are unchanged when the forecast_period bounds
+        match. """
+        cubeA = self.cube.copy()
+        add_forecast_reference_time_and_forecast_period(cubeA)
+        cubeB = self.cube.copy()
+        add_forecast_reference_time_and_forecast_period(cubeB)
+        bounds = [0., 1.]
+        cubeA.coord('forecast_period').bounds = bounds
+        cubeB.coord('forecast_period').bounds = bounds
+        result = iris.cube.CubeList([cubeA, cubeB])
+        _equalise_coord_bounds(result)
+        self.assertArrayAlmostEqual(result[0].coord('forecast_period').bounds,
+                                    [bounds])
+        self.assertArrayAlmostEqual(result[1].coord('forecast_period').bounds,
```

Comment:
If we are expecting the values to be literally unchanged, then we could use assertArrayEqual instead.